poppy/optics.py

```diff
@@ -804,11 +840,11 @@ def __init__(self, name=None, radius=1.0, pad_factor=1.0, **kwargs):
         # for creating input wavefronts - let's pad a bit:
         self.pupil_diam = pad_factor * 2 * self.radius
 
-    def getPhasor(self, wave):
+    def get_transmission(self, wave):
         """ Compute the transmission inside/outside of the occulter.
         """
         if not isinstance(wave, Wavefront):  # pragma: no cover
-            raise ValueError("CircularAperture getPhasor must be called with a Wavefront "
+            raise ValueError("CircularAperture get_opd must be called with a Wavefront "
```

Comment:
Somewhere I had started on a decorator that enforced `wave` is-a `Wavefront` subclass. I should finish that...